### PR TITLE
column mapping: update readme

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 
column mapping's description in readme is wrong

### What is changed and how it works?
update column mapping's readme

### CODE REVIEW

**Review by csuzhangxc — DISMISSED** [review]
LGTM

File: pkg/column-mapping/README.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ The separator argument defaults to an empty string if omitted.
 
 ## notice
 * only support above poor expressions now
-* column mapping don't change column type and table structure now
\ No newline at end of file
+* column mapping don't change column type and table structure now
```
Comment (amyangfei) [inline pkg/column-mapping/README.md]:
```suggestion
* column mapping doesn't change column type and table structure now
```

**Review by amyangfei — COMMENTED** [review]
Rest LGTM

**Review by amyangfei — APPROVED** [review]
LGTM

**Pull request merged**